src/assets/styles/mixins/breakpoints.scss

```diff
@@ -1,10 +1,19 @@
+// Global Breakpoints
+$breakpoint-laptop: 1024px;
```

Comment:
We already had breakpoints based on material in `scss-imports/cssvars`.
Need to consolidate with them in some way.